service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterReportBundleService.java

```diff
@@ -27,7 +27,7 @@ public Bundle createEncounterBundle(POCDMT000002UK01ClinicalDocument1 clinicalDo
         Encounter encounter = encounterMapper.mapEncounter();
 
         addEncounter(bundle, encounter);
-        addEpisodeOfCare(bundle, encounter);
+        addServiceProvider(bundle, encounter);
```

Comment:
why here the episode of care has been removed? Adding service provider seems ok.

Reply:
Episode of care is a more complex element and I had troubles creating a valid message - so I replaced it with a simple one which has no mandatory elements.